docs/reference/filters.md

```diff
@@ -446,11 +446,15 @@ Parameters:
 * content (string)
 * content type (string) - optional
 
-Example:
+content may contain [template placeholders](#template-placeholders).
+If a template placeholder can't be resolved then empty value is used for it.
```

Comment:
Maybe you should add that using request data in the response is prone to injections and can lead to refletced XSS, HTML injection, ...